WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -245,10 +232,13 @@ extension WPStyleGuide {
         let followIcon = Gridicon.iconOfType(.readerFollow, withSize: size)
         let followingIcon = Gridicon.iconOfType(.readerFollowing, withSize: size)
 
-        let tintedFollowIcon = followIcon.imageWithTintColor(.primary(shade: .shade40))
-        let tintedFollowingIcon = followingIcon.imageWithTintColor(.success(shade: .shade50))
+        let normalColor = UIColor.primary
+        let highlightedColor = UIColor.primaryDark
+        let selectedColor = UIColor.success
 
-        let highlightIcon = followingIcon.imageWithTintColor(.primaryLight)
+        let tintedFollowIcon = followIcon.imageWithTintColor(normalColor)
```

Comment:
Is there a reason why these (and the other reader icons) are set as tinted images rather than setting the button tint color?

Reply:
Only because I'm trying to make minimal changes on the frozen branch. Thoughts?

Reply:
Okay, sure - can we make a note to maybe look at updating that next time?